cmd/kpod/diff.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/containers/storage/pkg/archive"
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+type diffJSONOutput struct {
+	Changed []string `json:"changed,omitempty"`
+	Added   []string `json:"added,omitempty"`
+	Deleted []string `json:"deleted,omitempty"`
+}
+
+var (
+	diffFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:   "archive",
+			Usage:  "Save the diff as a tar archive",
+			Hidden: true,
+		},
+		cli.BoolFlag{
+			Name:  "json",
+			Usage: "Format output as JSON",
+		},
+	}
+	diffDescription = fmt.Sprint(`Displays changes on a container, layer, or image's filesystem.  The
```

Comment:
I am a user of docker, I have no idea what a layer is.  Currently this command can only tell me changes to a containers file system.  In the future, I think you will give me patches that show me the differences between multiple images. But for now lets keep it to what the tool does.